app/models/duty_calculator.rb

```diff
@@ -34,7 +33,10 @@ def additional_duty_rows
         next if option_klass.nil?
         next if measure.all_duties_zero?
 
-        acc << option_klass.new(measure, [], nil).option
+        option = {}
+        option[:key] = option_klass.id
+        option[:evaluation] = option_klass.new(measure, [], nil).option
```

Comment:
Made the additional options consistent with the other types of duty option. Could move this process to a shared builder at some point (another PR)